extensions/default/src/ViewerLayout/constants/panels.ts

```diff
@@ -0,0 +1,34 @@
+const expandedInsideBorderSize = 4;
+const collapsedInsideBorderSize = 4;
+const collapsedOutsideBorderSize = 8;
+
+const panelGroupDefinition = {
```

Comment:
Is there really a need for this file, or can we simplify things by putting this code directly in ResiablePanelsHook? If it's used in multiple places, that's a pretty good reason to keep it as a separate file, but if not, it's kind of redundant.

Reply:
we discussed this on slack and we decided to keep it. I personally think it's too many lines and large enough to be separate.